GagarinDmitry/src/Services/Basket/Basket.Domain/Entities/BaseEntity.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Basket.Domain.Entities
+{
+    public abstract class BaseEntity
+    {
+        public int Id { get; set; }
```

Comment:
How many different values can int store? In your opinion, is it enough?

Reply:
well, I think if I need to ensure uniqueness at the table level, then int(2 147 483 647) is enough for me

Reply:
Ok) Imagine the situation: In the near future you service will become very popular. 50% of the Earth desire to use your service to watch films. Is it still enough?

Reply:
well then, yes, I need to use Guid, but if we take this solution as a test, then why not,?

Reply:
Of course it is up to you to make decision about using int or Guid, but I advise you to treat your project as real product